frontend/src/auth.module.css

```diff
@@ -0,0 +1,126 @@
+.app {
+    text-align: center;
+    color: #38023b;
+    font-weight: bold;
+    font-size: 36px;
+  }
+  .authType{
```

Comment:
looks like everything here and below are indented an extra level so let's fix that
also, for CSS the preference is hyphens rather than camel case, so let's rename this classname to `.auth-type` and do the same for all of these below